src/Modules/AuthModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,5 +1,95 @@
-import React from "react";
+import { useForm } from "react-hook-form";
+import logo from "../../../../assets/images/PMS 3.png";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+import { useToast } from "../../../../Context/ToastContext";
+import { FormData } from "../../../../interfaces/Auth";
 
 export default function ForgetPass() {
-  return <>ForgetPass</>;
+	const [spinner, setSpinner] = useState<boolean>(false);
+	const { showSuccessToast, showErrorToast } = useToast();
+
+	const {
+		register,
+		handleSubmit,
+		formState: { errors },
+	} = useForm<FormData>();
+	const navigate = useNavigate();
+
+	const onSubmit = async (data: FormData) => {
+		setSpinner(true);
+
+		try {
+			const response = await axios.post(
+				`https://upskilling-egypt.com:3003/api/v1/Users/Reset/Request`,
+				data
+			);
+			showSuccessToast(
+				"Your request is being processed, please check your email"
+			);
+			navigate("/reset-pass");
+		} catch (error) {
+			if (axios.isAxiosError(error) && error.response) {
+				showErrorToast(error.response.data.message);
+			} else {
+				// Handle other types of errors here
+				showErrorToast("An error occurred.");
+			}
+		} finally {
+			setSpinner(false);
+		}
+	};
+
+	return (
+		<>
+			<div className="Auth-container vh-100 row align-items-center justify-content-center overflow-auto gx-0 flex-nowrap ">
+				<div className="logo  col-md-5 text-center">
+					<img src={logo} alt="logo" className="mb-3" />
+				</div>
+
+				<div className="login-container col-md-5 col-sm-10 rounded-4 px-5 py-5">
+					<p className="text-white m-0">Welcome to PMS</p>
+					<h3 className="color-text mb-4 forget-pass">Forget Password</h3>
+					<form onSubmit={handleSubmit(onSubmit)}>
+						<div className="form-group">
+							<label className="color-text" htmlFor="exampleFormControlInput1">
+								Email
+							</label>
+							<input
+								id="exampleFormControlInput1"
+								type="email"
+								className="form-control m-1 px-2"
+								{...register("email", {
```

Comment:
Email, password and other validation will be used in multiple places. I consider making a file containing all needed validations and then use this validation in our application. This will make our code more concise. 

You can look at this example:
https://github.com/AnasKhaled7/PMS-React/blob/main/src/lib/InputValidator.ts